_posts/2021-03-19-status-update.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: "2021-03-19: IAC Team Status Update"
+author: pmcmaw
+categories:
+  - team
+  - status
+  - community
+  - holidays
+  - releases
+  - modules
+  - gems
+  - ci
+tags:
```

Comment:
```suggestion
tags:
  - community
  - holidays
  - releases
  - modules
  - gems
  - ci
```

Reply:
Thanks, I literally have no words for what I done there. :-|

Reply:
it happens :shrug: